toolchain/check/context.h

```diff
@@ -774,6 +776,8 @@ class Context {
   // The stack of generic regions we are currently within.
   GenericRegionStack generic_region_stack_;
 
+  InstBlockStack vtable_stack_;
```

Comment:
Can you add a comment to document this and its semantics? Like I don't know if this is right:

```suggestion
  // Contains a vtable block for each `class` scope which is currently being defined, regardless of whether the class can have virtual functions.
  InstBlockStack vtable_stack_;
```